var/spack/repos/builtin/packages/glib/package.py

```diff
@@ -19,7 +19,7 @@ class Glib(Package):
     """
 
     homepage = "https://developer.gnome.org/glib/"
-    url = "https://ftp.gnome.org/pub/gnome/sources/glib/2.53/glib-2.53.1.tar.xz"
+    url = "https://download.gnome.org/sources/glib/2.53/glib-2.53.1.tar.xz"
```

Comment:
FWIW. Technically this property isn't needed when `url_for_version` is implemented.